src/slack/config.go

```diff
@@ -118,6 +118,31 @@ var stopDialog = dialog{
   }`),
 }
 
+// redeployDialog is the format of the menu that will displayed for redeploying services
+var redeployDialog = dialog{
+	name: "redeploy",
+	content: []byte(`{
+    "callback_id": "redeploy-xxxx",
+    "title": "Redeploy service",
```

Comment:
We are using `App` in all other dialogs. Probably change them to `Service` also.